Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -168,10 +176,20 @@ private class ParameterSetNames
         /// </para>
         /// </summary>
         [Parameter(Position = 2, Mandatory = true, ParameterSetName = ParameterSetNames.ContentsFromFile)]
+        [ValidateNotNullOrEmpty()]
```

Comment:
Yes, so PowerShell will automatically check for us that the value given is not null or empty.